### ISSUE TITLE
Save state prompt to avoid accidental overwrite

### ISSUE THREAD

**Issue by guirossibrum** [c3d/DB48X-on-DM42#1012]
I have unwillingly saved the calculator state a couple of times because I had double shift pressed when I clicked EXIT. I suggest the user to be prompted if the state will be overwritten yes/no. 
thanks

**Comment by c3d** [comment]
Good idea, but I will make that conditional under a flag, because for me, being able to save very quickly is a feature, not a bug.
